python/src/scipp/plotting/resampling_model.py

```diff
@@ -120,22 +123,32 @@ def _call_resample(self):
             self._home = self._resampled
             self._home_params = self._resampled_params
 
+    def update_array(self, array):
+        """
+        Update the internal array with a new array.
+        """
+        self._array = array
+
+    def reset_params(self):
+        """
+        Reset view parameters to force a plot update.
+        """
+        self._home_params = None
+        self._resampled_params = None
```

Comment:
Is `reset()` a better name? The client does not really know or have to know that it is params that are being reset?